modules/javafx.controls/src/main/java/com/sun/javafx/scene/control/VirtualScrollBar.java

```diff
@@ -129,18 +129,34 @@ public final BooleanProperty virtualProperty() {
         if (isVirtual()) {
             adjusting = true;
             double oldValue = flow.getPosition();
+            double newValue = ((getMax() - getMin()) * Utils.clamp(0, pos, 1)) + getMin();
+            /**
+             * JDK-8173321: Click on trough has no effect when cell height > viewport height:
+             * Solution: Scroll one cell further up resp. down if only one cell is shown.
```

Comment:
We generally do not refer to specific bug IDs in source code comments. Rather, describe the purpose of this block. Also, please spell out `resp.` -- I can't figure out what it is supposed to mean.

Reply:
I will change the comment in my next commit to:
Scroll one cell further in the direction the user has clicked if only one cell is shown.
Otherwise, a click on the trough would have no effect when cell height > viewport height.